templates/cicd/pipeline_cfn.yml

```diff
@@ -7,7 +7,20 @@
 # limitations under the License.
 AWSTemplateFormatVersion: '2010-09-09'
 Description: CodePipeline for {{$.AppName}}
+Parameters:
+  ProviderType:
+    Type: String
+    Default: {{.Source.ProviderName}}
+Conditions:
+  IsBitbucket:
```

Comment:
Do we need this condition? I thought we can just use `if eq .Source.ProviderName "Bitbucket"` to be consistent with the stages below.

Reply:
Yup, I actually just changed this to use go templates bc I was getting an error when it tried to call the new ConnectionName method for non-CodeStar sources. Thanks!